src/App.js

```diff
@@ -3,20 +3,35 @@ import Home from './pages/Home';
 import { Link, Route, Routes } from 'react-router-dom';
 import Products from './pages/Products';
 import Product from './pages/Product';
-//import { Navbar, NavbarHeader, NavbarItems, NavItem } from 'react-bootstrap-navbar';
+import { ThemeContext, themes } from './context/themeContext';
+import { useState } from 'react';
+import ToggleSwitch from './components/ToggleSwitch';
 
 function App() {
+  const [theme, setTheme] = useState(themes.dark);
+
+  const toggleTheme = () => {
+    setTheme((prevValue) =>
+      prevValue === themes.dark ? themes.light : themes.dark
+    );
+  };
+
   return (
-    <div className="App">
-      <Link to="/">Home</Link>
-      <Link to="/products">Products</Link>
-      <hr /> 
-      <Routes>
-          <Route path="/" element={<Home/>} />
-          <Route path="/products" element={<Products/>} />
-          <Route path="/products/:product_id" element={<Product/>} />
-      </Routes>
-    </div>
+    <ThemeContext.Provider value={{ theme: theme}}>
```

Comment:
Don't forget about the refactoring section of the context api https://neit.instructure.com/courses/53221/pages/live-coding-ii-refactoring-opportunities?module_item_id=2248881. The link outlines the preferred approach